models/cam-fv/camdart_obs_preprocessor.rst

```diff
@@ -0,0 +1,122 @@
+   +--------------+--------------------+--------------------------------------------------------------------------------+
+   | filename_out | character(len=256) | Observation sequence file to create and write. If this file exists it will be  |
+   |              |                    | overwritten.                                                                   |
+   +--------------+--------------------+--------------------------------------------------------------------------------+
+   | verbose      | logical            | If true, write more debugging/status information.                              |
+   |              |                    |                                                                                |
+   +--------------+--------------------+--------------------------------------------------------------------------------+
+   | calendar     | character(len=32)  | The string name of a valid DART calendar type. (See the                        |
+   |              |                    | :doc:`../../modules/utilities/time_manager_mod` documentation for a list of    |
+   |              |                    | valid types.) The setting here does not change what is written to the output   |
+   |              |                    | file; it only changes how the date information is printed to the screen in the |
+   |              |                    | informational messages.                                                        |
+   +--------------+--------------------+--------------------------------------------------------------------------------+
+   | print_every  | integer            | If set to a positive integer, print a status message each time after           |
+   |              |                    | processing that number of observations.                                        |
+   +--------------+--------------------+--------------------------------------------------------------------------------+
+
+| 
+
+Discussion
+----------
+
+See the documentation in the obs_kind and obs_def modules for things you can query about an observation, and how to set
+(overwrite) existing values.
+
+| 
+
+Building
+--------
+
+This program will be built when ``models/cam-fv/work/quickbuild.sh`` is run.
+
+The ``preprocess`` program must be built and run first to define what set of observation types will be supported. See
+the :doc:`../../../assimilation_code/programs/preprocess/preprocess` for more details on how to define the list and run
+it. The ``&preprocess_nml`` namelist in the ``input.nml`` file must contain files with definitions for the combined set
+of all observation types which will be encountered over all input obs_seq files.
+
+If you have observation types which are not part of the default list in the &preprocess_nml namelist, add them to the
+input.nml file and then run quickbuild.sh.
+
+
+Files
```

Comment:
Also list caminput.nc and cam_phis.nc, since those are needed by static_init_model?  (Or whatever names are put into model_mod_nml:cam_template_filename and cam_phis_filename)

Reply:
those files aren't required when building the program but they will be required when it's run.   if there's a 'running' section they'd go there, or just a mention in the 'Files' section maybe?